test/loader.test.js

```diff
@@ -35,6 +35,11 @@ describe('loader.js', () => {
             ]
         };
 
+        it('accepts url rules', () => {
+            const url = loader.loadRules(['https://raw.githubusercontent.com/wework/speccy/master/rules/default.json']);
```

Comment:
My concern about this test is that it goes out to the actual internet. If github is down, the test suite will fail. If a contributor is offline, the test suite will fail.

To get around this most languages/frameworks have some sort of "web mock" library. In ruby it's literally called webmock, and in Node there's an awesome one called [nock](https://github.com/node-nock/nock). 

We could maybe do it with express, but nock might be a tad lighter, and avoid any awkward firewall related issues that could come from actually running a server locally.